wg-releases/wg-qa/README.md

```diff
@@ -21,16 +21,23 @@ Oversees stability, quality, and testing criteria within Electron.
 
 ## Current Objective and Key Results
 
-**Objective:**
+**Objective 1:**
 
-Electron stable releases should be of high quality and easy for apps to upgrade.
+Early detection of Electron issues
 
 **Key Results:**
-* Measure the average or cumulative delta between an electron stable release, and the AFP participants shipping that same stable release in their GA product.
-* Measure the average or cumulative delta between an electron stable release, and the AFP participants shipping that same stable release in their canary product.
-* Reduce the size of outstanding AFP-raised issues at the time of stable.
-* Reduce the number of _would-be blockers_ discovered _after_ a stable ship.
-* Add 4 new members and 1 new organisation/company.
+
+* Report alpha/beta electron issues on latest 2 release versions and beta version (as a team)
```

Comment:
What is the output of this KR?  Is it going to be a document or a page with all the versions for the latest 2 released and all the issues for each one?
How do we consider the KR complete? There can always be a new bug we didn't find before, right?

Reply:
This is likely an evergreen KR - as a team, we report on and discuss issues in the monthly team syncs as well as the weekly reports in channel to give others a heads up, which has worked well several times.
There can definitely be a bug we didn't find before. I'd like to see what others think of the output/completion of this KR.

Reply:
Because the team already has a workflow in place, maybe this KR should be replaced by another one? Or maybe it could be a new way of reporting that we believe will be more effective?

My issue is that with the current framing and what you've told me, what is going on is already a "given". We are not making any progress and just maintaining the status quo.

Does it make sense?

Reply:
~~* Report alpha/beta electron issues on latest 2 release versions and beta version (as a team)~~
* At least one app to integrate Sentinal

Reply:
hey folks! i'm excited yall want to try out my ongoing work, but i definitely want to make sure we're aligned on requirements and restraints of the project and what an app would need to be able to report back to the service in order to get the most out of it, since it is in alpha and not in a state where there's available documentation for said constraints.

What's the timeframe for this being merged? Perhaps i should visit the next QA meeting?

Reply:
@codebytere our next meeting is 7/6. We were going to merge like tomorrow'ish